src/listeners/ticket/create-modal-submit-validator.ts

```diff
@@ -0,0 +1,183 @@
+import { ApplyOptions } from '@sapphire/decorators';
+import { Events, Listener } from '@sapphire/framework';
+import {
+    ActionRowBuilder,
+    ButtonBuilder,
+    ButtonStyle,
+    ChannelType,
+    EmbedBuilder,
+    GuildMember,
+    PermissionFlagsBits,
+    type Interaction,
+    type ModalSubmitInteraction,
+} from 'discord.js';
+import { Color } from '../../lib/embeds';
+import { TButtonId, InputId, ModalId } from '../../lib/ticket';
+import { messageLogger } from '../..';
+import { settingsManager } from '../..';
+import { userUrl } from '../../lib/user';
+import { ticketLogs } from '../..';
+import type { TicketLog } from '../../lib/ticket';
```

Comment:
The 2 imports from `'../../lib/ticket'` should be combined:
```suggestion
import { InputId, ModalId, TButtonId, type TicketLog } from '../../lib/ticket';
```